Stanford/JumpstartEngineering/Install/CAPx/CAPxDisplay.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * @file
+ * Define the context for the CAP display view mode for person pages.
+ */
+
+namespace Stanford\JumpstartEngineering\Install\CAPx;
+/**
+ *
+ */
+class CAPxDisplay extends \AbstractInstallTask {
```

Comment:
I definitely want dependencies listed in the .info file. But the order-and-harmony-in-the-universe side of me thinks that, since this is a separate thing from the install profile, and can be used elsewhere, dependencies should be declared in this class.